pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/service/admin/UserAdminTestUtil.java

```diff
@@ -135,11 +203,12 @@ public int setStudyAdminAndRoles(String adminEmail, String adminRole, List<Strin
 
     /**
      * Add roles for study
+     *
      * @param roles actual DSM roles
      */
     public void addStudyRoles(List<String> roles) {
         assertInitialized();
-        for (String role: roles) {
+        for (String role : roles) {
```

Comment:
Why this change and other similar changes?

Reply:
I think my checkstyle fixes actually broke this file. I'll revert it and fix my checkstyle later